JustSaying/JustSayingFluently.cs

```diff
@@ -385,4 +404,10 @@ public IMayWantOptionalSettings WithAwsClientFactory(Func<IAwsClientFactory> aws
             return this;
         }
     }
+
+    public enum TopicQueryBehaviour
+    {
+        DisableExistenceCheck,
+        NoPreload

```

Comment:
Agreed that `TopicQueryBehaviour` has an implicit 3rd value i.e "preload" that should be shown. Options are basically
- Preload, and use that list to check
- Don't preload, and check with new data each time
- Don't preload, and don't check at all.

What's good naming here?

And that the line `skipTopicPreload = true;`  is better as `needsTopicPreload = false;`